src/runtime/entitlements-manager.js

```diff
@@ -144,12 +132,6 @@ export class EntitlementsManager {
       if (response.isReadyToPay != null) {
         this.analyticsService_.setReadyToPay(response.isReadyToPay);
       }
-      if (
-        this.config_.analyticsMode == AnalyticsMode.IMPRESSIONS ||
-        this.isGoogleUtmSource_()
```

Comment:
I believe this was the only place this function was being used.  Should we remove it?

Reply:
Yes, good catch.